src/app/legacy/components/Promo/index.test.jsx

```diff
@@ -77,6 +109,62 @@ describe('Promo component - Image', () => {
       expect(srcSet).toContain(`${resolution}w`);
     });
   });
+
+  it('should append a webp file extension - programmes iChef recipes', () => {
```

Comment:
Do we need a test case to check that it _doesn't_ append webp when it is already in the image URL?

Reply:
This tes 

` it('should have a webp file extension - programmes iChef recipes', () => {
    render(<FixtureProgrammesWithWebp />);

    const image = screen.getByAltText('Test image alt text');
    const srcSet = image.getAttribute('srcSet');

    const expectedResolutions = [96, 128, 176, 240, 352, 464, 672, 800];
    expectedResolutions.forEach(resolution => {
      expect(srcSet).toContain(
        `https://ichef.bbci.co.uk/images/ic/${resolution}xn/p06vzdgj.jpg.webp ${resolution}w`,
      );
    });
  });
`

should do that as the fixture data already has .webp at the end, and then when it checks the urls with resolution, it will fail if it has two .webps as it won't match that substring.

But I think  the test description could be clearer to show that, so I'll change that. Thanks.